### PR TITLE
#2320 move formatted message out of interfaces

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**
this PR 
- removes the last occurrences of old Money from the code. The #2877 is solved (there's still some cleanup needed though)
- moves all `FormattedMessage`s and `React` dependency out of interfaces.

### CODE REVIEW

File: app/components/shared/formatters/MoneyRange.tsx [hunk 1]
```diff
@@ -56,17 +71,20 @@ export const MoneyRange: React.FunctionComponent<
...
-      ) : (
+      );
+    } else {
+      formattedValue = (
         <FormatShortNumberRange
           valueFrom={valueFrom}
           valueUpto={valueUpto}
           inputFormat={inputFormat}
-          outputFormat={outputFormat}
+          outputFormat={outputFormat as EAbbreviatedNumberOutputFormat}
```
Comment (Moejoe90) [inline app/components/shared/formatters/MoneyRange.tsx]: why do we need to do the force casting here?

File: app/components/shared/formatters/Money.tsx [hunk 2]
```diff
@@ -70,39 +71,42 @@ const MoneyNew: React.FunctionComponent<IMoneyProps & IMoneyCommonProps & Common
     //todo: this should pass through 0 as well. Use isValidNumber from the #2687 PR when it's merged
     const decimalPlaces = selectDecimalPlaces(moneyFormat, outputFormat);
     if (
-      Object.values(ENumberOutputFormat).includes(outputFormat) &&
-      (outputFormat === ENumberOutputFormat.FULL_ROUND_UP ||
-        outputFormat === ENumberOutputFormat.ONLY_NONZERO_DECIMALS_ROUND_UP)
+      outputFormat === EAbbreviatedNumberOutputFormat.SHORT ||
```
Comment (Moejoe90) [inline app/components/shared/formatters/Money.tsx]: Do you have unit tests for all these cases?

File: app/components/shared/formatters/MoneyRange.tsx (hunk 1 above)
Comment (AlexeyTuboltsev) [reply app/components/shared/formatters/MoneyRange.tsx]: done. The reason was that ts doens't understand `Object.values(ENumberOutputFormat).includes(outputFormat)` on enums, mind you

File: app/components/shared/formatters/Money.tsx (hunk 2 above)
Comment (AlexeyTuboltsev) [reply app/components/shared/formatters/Money.tsx]: i added a couple

**Pull request merged**